examples/fakeserver/README.md

```diff
@@ -0,0 +1,16 @@
+# StellarStation API Fakeserver
+
+A fake implementation of the StellarStation API to use to check behavior of client code before
+integrating with the service.
+
+## Features
+
+- Verifies a fake API key
+- Returns 1MB of random telemetry every second
+- Echos back received commands
+- Cancels the connection after 5 minutes
+- Only allows requests for satellite ID `"5"`, other IDs show behavior for non-existent satellites
+
+## Running
+
+// TODO(rag): Add instructions for running with docker after pushing an image.
```

Comment:
Examples should be as simple as possible so that users can focus on learning the APIs which is new to them. As a command to run the server, just using gradle task or Java command would better not to distract users by using additional technical stack like Docker, which could confuse them on what is doing or may need additional steps of installing docker.

For example, we could add execute task in gradle and give an instruction like this.

```
task execute(type:JavaExec) {
    main = 'com.stellarstation.api.fakeserver.FakeServerMain'
    classpath = sourceSets.main.runtimeClasspath
}
```

> Run following command at the top directory of the repository in order to launch the fake server.
> $ ./gradlew examples:fakeserver:execute

Reply:
Thanks - there is already a `run` task which indeed I should document here. I'll also document docker since the intention is people can run the fakeserver without cloning this git repository during their initial development phase.